extensions/community_extensions/dsa_transparency.md

```diff
@@ -0,0 +1,345 @@
+<ul>
+<li>That the ad is indeed an ad;</li>
+<li>The identity of the advertiser;</li>
+<li>The identity of the party that financed the ad, if it is different from the advertiser;</li>
+<li>Information about the &ldquo;main parameters&rdquo; used to select the ad presented to the end-user;</li>
+<li>Where applicable, information about any means users may have at their disposal to change those main parameters.&nbsp;&nbsp;</li>
+</ul>
+<p>Although the legal obligation to provide the user-facing information disclosures applies to Online Platforms, it is clear that in many advertising scenarios, those platforms will need to rely on third-party vendors for the data that will be required to populate the disclosures.&nbsp; To ensure that the third parties are equipped to provide this support, this technical specification standardizes the collection, compilation and transport of the data, leaving Online Platforms free to decide how they wish to make the user-facing disclosures, including if they want to delegate the disclosures to another party.&nbsp;&nbsp;</p>
+<p>This technical specification provides data formats and transport for the advertising industry to implement relevant DSA transparency information. This solution should be applicable across most relevant use cases including; programmatic and non-programmatic media buys, channels including desktop web, mobile (web/app), video, CTV.</p>
+<h2>Object Specification for OpenRTB 2.X</h2>
+<h3>Bid Request</h3>
+<h4>Object: Regs</h4>
+<div>
+<table>
+<tbody>
+<tr>
+<td><strong>Attribute</strong></td>
+<td><strong>Type</strong></td>
+<td><strong>Description</strong></td>
+</tr>
+<tr>
+<td><code>ext.dsa</code></td>
+<td>object&nbsp;</td>
+<td>Extension for DSA transparency information</td>
+</tr>
+</tbody>
+</table>
+</div>
+<h4>Object: DSA</h4>
+<div>
+<table>
+<tbody>
+<tr>
+<td><strong>Attribute</strong></td>
+<td><strong>Type</strong></td>
+<td><strong>Description</strong></td>
+</tr>
+<tr>
+<td><code>dsainfo</code></td>
+<td>integer</td>
+<td>Flag to indicate if DSA information should be made available. This will signal if the bid request belongs to an Online Platform/VLOP, such that a buyer should respond with DSA Transparency information based on the pubrender value.
+    <p><code>0</code> = Not required</p>
+    <p><code>1</code> = Supported, bid responses with or without DSA object will be accepted</p>
+    <p><code>2</code> = Required, bid responses without DSA object will not be accepted</p>
+    <p><code>3</code> = Required, bid responses without DSA object will not be accepted, Publisher is an Online Platform</p></td>
+</tr>
+<tr>
+<td><code>pubrender</code></td>
+<td>integer</td>
+<td>Flag to indicate if the publisher will render the DSA Transparency info. This will signal if the publisher is able to and intends to render the icon and display the DSA transparency info to the end user.
```

Comment:
Could a publisher's ability to render depend on a specific `Imp` object, in case multiple impressions are offered through a single bid request? If that's the case, should this flag reside somewhere under `Imp` object?

For example… a publisher might be able to render the transparency information for native ads, but not for HTML banners.

Reply:
Similar to the comment above, the GPWG discussed and would like to tackle this post-MVP which this extension is meant to be.